src/app/todo-list/components/todo-list.component.ts

```diff
@@ -0,0 +1,138 @@
+import { Component, OnInit } from '@angular/core';
+import { StorageService } from '@app/service/storage.service';
+import { PlayerStateService } from '../../player/player-state.service';
+import {
+  Player,
+  MilestoneStatus,
+  MileStoneName,
+  InventoryItem,
+  Character,
+  NameQuantity,
+  SaleItem
+} from '@app/service/model';
+import { filter } from 'rxjs/operators';
+import { from, Observable } from 'rxjs';
+import { BungieService } from '@app/service/bungie.service';
+
```

Comment:
A lot of the code in here was added when I was going to try to integrate this more closely with the rest of the app. Since you recommended keeping it more isolated, that makes it easier for me to knock out this boilerplate. I'll go back and define some interfaces/models based on what I need. I'm still thinking if I want to re-use any interfaces or models you have defined. I definitely don't need all the fields on the models you have, and I would be adding more fields that you probably wouldn't use. It's not like it's a bad thing, but it could be worth it to keep them separate so that our work doesn't collide as much.